lib/Net/SIP/Simple/Call.pm

```diff
@@ -452,9 +452,17 @@ sub dtmf {
 	    $type = $self->{param}{sdp_peer}
 		&& $self->{param}{sdp_peer}->name2int('telephone-event/8000','audio');
 	} elsif ( $m eq 'audio' ) {
-	    $type = $self->{param}{sdp_peer}
-		&& $self->{param}{sdp_peer}->name2int('PCMU/8000','audio')
-		|| 0; # default id for PCMU/8000
+	    my $sdp_peer = $self->{param}{sdp_peer};
+	    if ($sdp_peer) {
+		$type = $sdp_peer->name2int('PCMU/8000','audio') unless defined $type;
```

Comment:
Since Net::SIP declares to require at least Perl 5.10 one could simply write

       $type //= $sdp_peer->name2int('PCMU/8000','audio')

Reply:
Done!